packages/react-ui/components/Checkbox/__stories__/Checkbox.stories.tsx

```diff
@@ -154,6 +157,36 @@ Plain.story = {
             .perform();
           await this.expect(await this.takeScreenshot()).to.matchImage('spacePress');
         },
+        async spacePressAfterClick() {
+          await this.browser
+            .actions({
+              bridge: true,
+            })
+            .click(this.browser.findElement({ css: '#test-element' }))
+            .perform();
+          await this.browser
+            .actions({ bridge: true })
+            .sendKeys(this.keys.SPACE)
+            .perform();
+          await this.expect(await this.takeScreenshot()).to.matchImage('spacePressAfterClick');
+        },
+        async doubleSpacePressAfterClick() {
+          await this.browser
+            .actions({
+              bridge: true,
+            })
+            .click(this.browser.findElement({ css: '#test-element' }))
+            .perform();
+          await this.browser
+            .actions({ bridge: true })
+            .sendKeys(this.keys.SPACE)
+            .perform();
+          await this.browser
+            .actions({ bridge: true })
+            .sendKeys(this.keys.SPACE)
```

Comment:
```suggestion
          await this.browser
            .actions({ bridge: true })
            .sendKeys(this.keys.SPACE, this.keys.SPACE)
```

Если надо нажать несколько клавиш, то можно их перечислить через запятую.